fabric-1.19/src/main/resources/dynmap.mixins.json

```
@@ -0,0 +1,19 @@
+{
+  "required": true,
+  "minVersion": "0.8",
+  "package": "org.dynmap.fabric_1_19.mixin",
+  "compatibilityLevel": "JAVA_16",
```

```suggestion
  "compatibilityLevel": "JAVA_17",
```
You now require JAVA_17 with 1.19 right ?